pkg/playground/react-demo-listing.jsx

```diff
@@ -93,10 +95,10 @@ export function showListingDemo (rootElement, rootElementSelectable, rootElement
         },
     ];
 
-    var addLink = <button className="pull-right link-button" onClick={handleAddClick}>Add Row</button>;
+    var addLink = <Button variant="link" isInline className="pull-right" onClick={handleAddClick}>Add Row</Button>;
 
     var rowAction = {
-        element: <button className="pf-c-button pf-m-secondary btn-control" onClick={handlePlayClick}><i className="fa fa-play" /></button>,
+        element: <Button variant="secondary" isInline className="btn-control" onClick={handlePlayClick}><PlayIcon /></Button>,
```

Comment:
That would deserve an aria-label but not a big deal for repost.

Reply:
Fixed. I needed to repost anyway to fix some tests and the merge conflict.